lib/handlers/user/index.js

```diff
@@ -19,6 +19,7 @@ require('util').inherits(SteamUser, EventEmitter);
 // Methods
 
 SteamUser.prototype.logOn = function(logOnDetails) {
+    this._logOnDetails = logOnDetails;
```

Comment:
This is a valid concern. I wonder how your SteamUser instance executed the handler not having had `logOn` called, but it's not disallowed to have two SteamUser instances, for instance.

>You might have to add the _logOnDetails property to this._client

No - handlers shouldn't add properties to SteamClient, as it might lead to conflicts.